tests/Google/Ads/GoogleAds/Lib/V7/GoogleAdsClientTest.php

```diff
@@ -132,4 +141,19 @@ public function testNullLinkedCustomerIdNotAppearInClientOptions()
             $googleAdsClient->getGoogleAdsClientOptions()
         );
     }
+
+    public function testGrpcChannelCredentialWhenGrpcChannelIsSecureInClientOptions()
```

Comment:
When GRPC channel is "NOT" secure?
I see you set the value here as false.

Reply:
Good catch!